docs/preview/features/powershell/azure-key-vault.md

```diff
@@ -33,3 +33,25 @@ PS> $accessPolicies = Get-AzKeyVaultAccessPolicies -KeyVaultName "my-key-vault"
 PS> $accessPolicies = Get-AzKeyVaultAccessPolicies -KeyVaultName "my-key-vault" -ResourceGroupName "my-resouce-group"
 # accessPolicies: {list: [{tenantId: ...,permissions: ...}]}
 ```
+
+## Set secret value from file into key vault
+
+Sets a secret certificate from a file as plain text in Azure Key Vault.
+
+| Parameter      | Mandatory | Description                                                                          |
+| -------------- | --------- | ------------------------------------------------------------------------------------ |
+| `FilePath`	 | yes       | The path to the file containing the secret certificate to add in the Azure Key Vault |
```

Comment:
Same here, always document least specific to most specific so vault, secret, path, expires